Tests/TurboNavigatorTests.swift

```diff
@@ -271,6 +284,14 @@ final class TurboNavigatorTests: XCTestCase {
 
 private class EmptyDelegate: TurboNavigationDelegate {
```

Comment:
I wasn't sure whether to change EmptyDelegate or not 🤔. `MockDelegate` didn't sound right, so I stuck with Empty. Happy to rename it.

Reply:
What about `TestNavigationDelegate`? That aligns with other similar classes in the turbo-ios test suite.